### ISSUE TITLE
Attributa cannot be controlled by NVDA profiles

### ISSUE THREAD

**Issue by vlegoff** [Andre9642/BrailleExtender#16]
### In summary

NVDA profiles cannot specify several different options of Attributa, which would be handy.

### Details

NVDA profiles are a very powerful way to customize most settings and to let the user decide in what circumstance apply them (whether manual or automatic).  In my case, for instance, I like to see spelling errors marked in Braille (with dot 7 and 8).  There are moments, however, when this is neither necessary nor helpful and I do need to see what dot 7 and 8 hold.

My solution was to create an NVDA profile to turn off, manually, Attributa (or at least, its "spelling error" setting).  In some Word documents I edit, it's great to have the spelling errors shown.  In other Word documents, this is not so useful.  So I'd like to be able to turn off Attributa by simply selecting an NVDA manual profile.  As far as Attributa is concerned, however, different configuration options cannot be stored in different NVDA profiles.  There might be another way to do it, as Attributa seems to come with its own system for profiles (although only automatic, as far as I could see), and while not knowing how NVDA merges configuration based on profiles, I would guess this would be a welcome feature even if, perhaps, a little specific.

Thank you for your great job on this add-on,

**Comment by Andre9642** [comment]
Hi @vlegoff! :)
Thank you for your suggestion. I'll see what I can do ASAP.
In the meantime, you can assign a gesture to disable/enable Attribra quickly. Please go in command gestures dialog (in preferences menu), then in BrailleExtender section.
Thanks again.

**Comment by vlegoff** [comment]
Ho, that's correct.  I knew I was most likely forgetting something.  Thank you for the tip!

**Comment by Andre9642** [comment]
Done in e2c17d8
